pkg/apis/autoscaling/register.go

```diff
@@ -228,7 +228,7 @@ var (
 	}
 	MaxScaleAnnotation = kmap.KeyPriority{
 		MaxScaleAnnotationKey,
-		GroupName + "/maxScale",
```

Comment:
These legacy keys should not be removed - there are older clients/documentation that has `/maxScale` and we should preserve those